Provider/VimeoProvider.php

```diff
@@ -61,11 +61,13 @@ public function getHelperProperties(MediaInterface $media, $format, $options = a
         $box = $this->getBoxHelperProperties($media, $format, $options);
 
         $params = array(
-            'src'         => http_build_query($player_parameters),
-            'id'          => $player_parameters['js_swf_id'],
-            'frameborder' => isset($options['frameborder']) ? $options['frameborder'] : 0,
-            'width'       => $box->getWidth(),
-            'height'      => $box->getHeight(),
+            'src'               => http_build_query($player_parameters),
+            'id'                => $player_parameters['js_swf_id'],
+            'frameborder'       => isset($options['frameborder']) ? $options['frameborder'] : 0,
+            'width'             => $box->getWidth(),
+            'height'            => $box->getHeight(),
+            'class'             => isset($options['class']) ? $options['class'] : false,
```

Comment:
would use an empty string here as default and then check for emptiness in the twig template.
I dont like mixing types

@core23 what do you think?